frontend/amundsen_application/static/js/pages/DashboardPage/constants.ts

```diff
@@ -8,3 +8,13 @@ export const EDIT_DESC_TEXT = 'Click to edit description in';
 
 export const LAST_RUN_SUCCEEDED = 'succeeded';
 export const STATUS_TEXT = 'Status:';
+
+export const TABLES_TAB_TITLE = 'Tables';
+export const CHARTS_TAB_TITLE = 'Charts';
+export const QUERIES_TAB_TITLE = 'Queries';
+
+export enum DASHBOARD_TAB {
+  TABLE = 'table',
```

Comment:
We changed from 'tables' to 'table', was this something we wanted?

Reply:
Changed back to `tables`. Should be consistent across all tabs showing tables now. Good catch.